client/src/components/menubar/index.tsx

```diff
@@ -266,249 +257,218 @@ class MenuBar extends React.PureComponent<MenuBarProps, State> {
+            cursor: "pointer",
           }}
-        >
-          <ButtonGroup className={styles.menubarButton}>
-            <AnchorButton
-              type="button"
-              icon={IconNames.INFO_SIGN}
-              onClick={() => {
-                dispatch({
-                  type: "toggle active info panel",
-                  activeTab: "Dataset",
-                });
-              }}
-              style={{
-                cursor: "pointer",
-              }}
-              data-testid="drawer"
-            />
-          </ButtonGroup>
-          {isDownload && (
-            <Tooltip
-              content="Download the current graph view as a PNG"
-              position="bottom"
-              hoverOpenDelay={globals.tooltipHoverOpenDelay}
-            >
-              <AnchorButton
-                className={styles.menubarButton}
-                data-testid="download-graph-button"
-                type="button"
-                icon={IconNames.CAMERA}
-                style={{
-                  cursor: "pointer",
-                }}
-                loading={screenCap}
-                onClick={() => dispatch({ type: "graph: screencap start" })}
-              />
-            </Tooltip>
-          )}
-          {isTest && (
-            <AnchorButton
-              className={styles.menubarButton}
-              type="button"
-              icon={IconNames.TORCH}
-              style={{
-                cursor: "pointer",
-              }}
-              data-testid={GRAPH_AS_IMAGE_TEST_ID}
-              data-chromatic="ignore"
-              loading={screenCap}
-              onClick={() => dispatch({ type: "test: screencap start" })}
-            />
-          )}
-          <Clip
-            // @ts-expect-error ts-migrate(2322) FIXME: Type '{ pendingClipPercentiles: any; clipPercentil... Remove this comment to see the full error message
-            pendingClipPercentiles={pendingClipPercentiles}
-            clipPercentileMin={clipPercentileMin}
-            clipPercentileMax={clipPercentileMax}
-            handleClipOpening={this.handleClipOpening}
-            handleClipClosing={this.handleClipClosing}
-            handleClipCommit={this.handleClipCommit}
-            isClipDisabled={this.isClipDisabled}
-            handleClipOnKeyPress={this.handleClipOnKeyPress}
-            handleClipPercentileMaxValueChange={
-              this.handleClipPercentileMaxValueChange
+          loading={screenCap}
+          onClick={() => dispatch({ type: "graph: screencap start" })}
+        />
+      </Tooltip>
+    ),
+    <AnchorButton
+      type="button"
+      icon={IconNames.INFO_SIGN}
+      onClick={() => {
+        dispatch({
+          type: "toggle active info panel",
+          activeTab: "Dataset",
+        });
+      }}
+      style={{
+        cursor: "pointer",
+      }}
+      data-testid="drawer"
+    />,
+  ];
+
+  let widthUnitIterator = 0;
+  let columnStartingIndex: null | number = null;
```

Comment:
iterator that counts the rendered width units and columnStartingIndex to mark which element is the first to be rendered in the column